qt/coda-qt.cpp

```diff
@@ -978,10 +1040,36 @@ QVariant Bridge::cool(const QString& messageStr)
     }
     else if (message == "uno .uno:CloseWin")
     {
-        // Close the main window associated with this web view
-        if (_webView && _webView->window())
+        // Close the tab if present, otherwise close the main window.
```

Comment:
(if you introduce terminating punctuation into a comment, my pet-peeve suggestion would be to use a trailing colon instead of a full-stop for comments that precede the commented-on code; that way, it becomes subtly more clear what the comment appertains to)